lib/Serialization/SerializeDoc.cpp

```diff
@@ -153,8 +153,16 @@ class DeclGroupNameContext {
     // We need the file path, so there has to be a location.
     if (VD->getLoc().isInvalid())
```

Comment:
`VD->getLoc(false)` works too.

Reply:
Oh yeah. I guess I'll leave it this way because eventually we'll want group info to work in non-single-frontend mode anyway.